### DISCUSSION TITLE
Is the call to `dup` necessary in `data.rb#L50`?

### DISCUSSION THREAD

**Discussion by stevegeek** [joeldrapper/literal#24] [General]
When copying the attributes hash in `#with` in Literal::Data a dup call creates a shallow copy of attributes, but then merge is called. But won't `#merge` create a new hash anyway? 

https://github.com/joeldrapper/literal/blob/main/lib/literal/data.rb#L50

**Comment by joeldrapper** [comment]
No, you're right. It's not necessary.
